utils/updatePackage.js

```diff
@@ -0,0 +1,86 @@
+var open = require("nodegit").Repository.open;
```

Comment:
How did I ever miss this question? :tongue:

I'm... not sure, other than in knowing that it was likely available to Atom users, I'm not sure. Perhaps I should have written coffeescript.